tests/regression/test_netgen.py

```diff
@@ -0,0 +1,269 @@
+from firedrake import *
+from mpi4py import MPI
+from netgen.geom2d import SplineGeometry
+from netgen.csg import CSGeometry, OrthoBrick, Pnt, Sphere
+from netgen.meshing import MeshingParameters
+from netgen.meshing import MeshingStep
+import netgen
+import numpy as np
+import gc
+from petsc4py import PETSc
+import pytest
+
+
+printf = lambda msg: PETSc.Sys.Print(msg)
+
+
+def poisson(h, degree=2):
+    comm = MPI.COMM_WORLD
```

Comment:
COMM_WORLD is already in the firedrake namespace (you get it from from firedrake import *)